nav-app/src/app/help/help.component.scss

```diff
@@ -12,6 +12,12 @@
     .content {
         overflow-y: scroll;
         max-height: 60vh;
+
+        span {
+            cursor:pointer;
+            color:blue;
+            text-decoration:underline;
+        }
```

Comment:
Probably this should also have a class so that spans used to apply other inline text effects won't look like hyperlinks. I don't think that happens right now but it would be future proofing for later additions to the help page. Also in #179 the markdown renderer may be more liberal in its application of <span>s (although this methodology may no longer be viable once #179 is implemented).